src/votekit/election_types.py

```diff
@@ -333,6 +395,17 @@ def random_transfer(
     return transfered
 
 
+def seqRCV_transfer(
+    winner: str, ballots: list[Ballot], votes: dict, threshold: int
+) -> list[Ballot]:
+    """
+    Useful for a Sequential RCV election which does not use a transfer method ballots \n
+    ballots: list of ballots \n
+    output: same ballot list
+    """
+    return ballots
```

Comment:
a comment might be useful to explain the use of this since it looks "wrong" at first glance, but has reasons to exist